---
aliases:
  - Bio
  - Biology
  - BIOL200 12-02-2024
tags:
  - "#BIOL200"
cssclasses:
---
From: [[2024/12/02]]

------
# Day 37 ([[12-02-2024]]) 

> 1 week and many things to do this week

unit 4 exam on Friday
	ch 11, 12, 13, 14, and parts of 15

keep up on readings

Today and Wednesday - Gene Expression

-----------------
# Dictionary
codon - 3 base
61/64

RNAP
- binds to promoter
- copies 1 strand into mRNA
- opens up double stranded DNA
- doesnt need a primer!

-------

Recall the Genetic code:

first base on left, second base on top, third base on right
	61 of 64 encode 20 amino acids
	3 of 64 encode stop codons
	1 of 64 encode stop codons (AUG)

# The genetic code has 7 main characteristics
...
...


# Code practically universal
- **Strongest evidence that all living things share common ancestry**
* Advances in genetic engineering
* Mitochondria and chloroplasts have some differences in "stop" signals

- Jellyfish GFP protein on pig snout and feet!

# Reading Frame
Strands
- Only one strand encodes protein for most genes
- Template strand: the strand that is used as a template during transcription
- Coding strand: The complementary strand
Start point for reading
	Because each codon is three bases long there are three possible start points when reading a coding strand or RNA sequence
Open Reading Frame
	String of codons un-interrupted by a "stop" codon

CGATCGATGCCGATCGATGTTAGCCTGAAGCCATTGAGGACTTTG
CCTAGCTACGGCTAGCTACAATCGGACTTCGGTAACTCCTGAAAC

Each strand has three possible coding frames depending on where you start "reading". Only one is "correct"

XXXTHEBIGREDCATATETHEBUGXXX
THE BIG RED CAT ATE THE BUG
HEB IGR EDC ATA TET HEB UG 
etc

The correct reading frame will be an "open reading frame" it will not have stop codons
5' CCG GCC AUG AUG UGU AAG CAG UUU GAU UGU AAA 3'
5' pro ala met met cys lys glu phe asp cys lys 3'

anytime we do this, we'll look for the start codon and go from there

# Predicting Amino Acid sequence
a) using the genetic code to predict...
...
...


# Basic Gene Structure
### Promoter
- Control region. NOT transcribed.
- Contains conserved regions
- Bases numbered negatively from right to left
### Transcription Unit
- Portion of gene transcribed into RNA
- Bases numbered positively from first base

![[Pasted image 20241202092205.png]]

Given:
5' ATG CCC TTT CCC TTT TAG 3'
3' is:
3' TAC GGG AAA GGG AAA ATC 5'
this is a very short open reading frame

RNA Polymerase = RNAP
a single RNAP in E Coli
	does all the transcribing
	has the ability to recognize and bind to promoter sequences upstream of genes

only the genes that cells need to make proteins need to be turned on
	RNAP binds to promoters to disable those that arent needed (i think?)
	
> #MEMORIZE Promoter is the place in the gene where RNA Polymerase binds and starts copying
> Only 1 strand gets copied

recall DNA gets unwound by Helicase before replication
RNA polymerase opens a little bubble and unwinds a template

after helicase, we need primase. It makes an RNA primer so that replication can get started
RNA polymerase doesnt need a primer, it just replicates!

transcription (an RNA copy) starts at +1

if DNA polymerase makes DNA in a 5' -> 3' direction
then RNA polymerase makes RNA in a 5' to 3' direction

> #Memorize RNA vs DNA:
> RNA has a 2' OH

Given:
5' ATG CCC TTT CCC TTT TAG 3'
3' TAC GGG AAA GGG AAA ATC 5' (from earlier!)

the bottom one gets copied (template strand)
the top one does NOT get copied (coding strand)
	this is so that the completed RNA runs 5' to 3'!
RNA sequence is:
5' AUG CCC UUU CCC UUU AUC 3' (coding strand but T's are U's bc RNA!)

template strand running 3' to 5' is the one that RNAP physically copies
coding strand running 5' to 3' contains the same code as new mRNA but with DNA nucleotides
#Memorize the difference

3 phases:
1. Initiation (or start)
	1. unwind DNA just a little bit
2. Elongation (or copy)
3. Termination (or finish)
	1. RNAP stops replication and lets go
	2. Marked by sequence that signals "stop" to polymerase

# Transcription Termination in Bacteria
- Newly synthesized RNA molecule forma a G-C rich *hairpin loop* followed by a run of U's

# Prokaryotic transcription is coupled to translation
- mRNA begins to be translated before transcription is finished
- Operon
	- grouping of functionally related genes
	- multiple ...
	- ...

recall transcription and translation in cytoplasm for prokaryotes
	-> coupled, happen at the same time
transcription in nucleus, translation in cytoplasm for eukaryotes
	-> not coupled, we dont do that!

# Initiation of transcription
- requires a *series* of transcription factors
- necessary to get the RNA polymerase II enzyme to a promoter and to initiate gene expression
- Interact with RNA polymerase ...
- ...
- ...

not coupled bc of the nucleus!

other stuff that has to happen:
the genes are structured differently in eukaryotes!
Three main modifications:
1. add a 5' cap
2. add a poly-A tail
3. splice out non-coding region
