---
aliases:
  - Bio
  - Biology
  - BIOL200 12-04-2024
tags:
  - "#BIOL200"
  - BIOL/CH15
cssclasses:
---
From: [[2024/12/04]]

------
# Day 38 ([[12-04-2024]]) 

![[Chapter 15 - Transcription, Translation, Mutation F24.pptx]]

no questions on transcription on unit 4 exam
	but on final!

extra credit review assignment posted!
	ton of Qs from the textbook to make sure we study
	"not exhaustive"
	will be another similar one for the final!

lowest of 4 exams gets dropped
	if final > lowest remaining exam, itll replace it!

------

# Euakryotic Transcription
- 3 different RNA polymerases
	- RNA polymerase I transcribes rRNA
	- ==RNA polymerase II== transcribes mRNA and some snRNA
	- RNA polymerase III transcribes tRNA and some other small RNAs
- Each RNA polymerase recognizes its own promoter

recall RNAP binds to a promoter
- promoter is non-coding!
- its the point that the RNAP starts unzipping
	- genes start right after it with a start codon

![[Pasted image 20241204091245.png]]

location of transcription and translation in prokaryotes and eukaryotes is fair game for a test question!
	prokaryotes: cytoplasm
	eukaryotes: nucleus -> cytoplasm

- Initiation of transcription
	- requires a _series_ of transcription factors
		- Necessary to get the RNA polymerase II enzyme to a promoter and to initiate gene expression
		- interact with RNA polymerase to form initiation complex at promoter
	- Eukaryotic RNA Pol cannot bind to mromoters on its own
- Termination
	- Termination sites not as well defined
- Transcription and Translation are NOT coupled in eukaryotes

![[Pasted image 20241204091506.png]]
#MEMORIZE euks -> 3 polymerases, transcription factors

# Elongation
- RNA pol II will often pause downstream of the promoter region
- Eloongation factors are used
- Machinery must deal with encountered nucleosomes

# Termination
- Different mechanism than prokaryotes
- Mechanism varies by polymerase
- Involves termination sequences/sites

recall:
heterochromatin -> highly condensed/compacted, not expressed bc RNAP cant find any promoters!

euchromatin -> expressed bc its less compacted!

# Post-Transcriptional Modifications
#MEMORIZE
- Eukaryotic mRNA is different than prokaryotic mRNA
- 5' Cap
	- protects against degradation
	- participates in initiation of translation
- 3' poly-A tail
	- protects against degradation
	- involved in nuclear export process
	- long string of Adenines (A's)
- splicing
	- removal of intervening sequences (introns)

![[Pasted image 20241204092215.png]]

introns vs exons
#MEMORIZE
introns -> non-expressed, spliced out (intervening)
exons -> expressed

spliceosome recognizes intron/exon junction, cuts them , and combines exons perfectly every time (black box magic)

- 5' cap
	- first nucleotide (often A or G) modified by the addition of GTP at the 5' position
	- methyl group added by guanylyl transferase -> methyl-G cap (occurs during transcription)
- poly A tail
	- transcript is cleaved prior to termination site
	- 100-200 adenines added by the enzyme poly-A polymerase
![[Pasted image 20241204092837.png]]
5' to 5' connection (the only one we know about!)
this is super helpful bc we have no enzyme that can break that bond! -> no degradation


# Eukaryotic pre-mRNA splicing
- Introns - non-coding sequences
- Exons - sequences that will be translated
- Small ribonucleoprotein particles (snRNPs) recognize the intron-exon boundaries
- snRNPs cluster with other proteins to form spliceosome
	- responsible for removing introns

![[Pasted image 20241204093006.png]]
![[Pasted image 20241204093012.png]]
![[Pasted image 20241204093020.png]]
need to know spliceosome, but not how it works
 need to know it cleaves exon and intron junctions and hooks exons together

introns float away, degrade, and get raw materials reused by the cell

# Alternative splicing

Single primary transcript can be spliced into different mRNAs by including different sets of exons
- Transcriptome: all the RNAs produced from a genome
- Proteome: all the proteins produced from a genome

Introns are spliced out of pre-mRNAs to produce the mature mRNA
Tissue-specific alternative splicing
Same gene makes calcitonin in the thyroid and calcitonin-gene related peptide (CGRP) in the hypothalamus
Determined by tissue-specific factors that regulate the processing of the primary transcript

![[Pasted image 20241204093334.png]]

# Effect of alternative splicing
Many human genetic disorders may be due to altered splicing
One estimate suggests 95% of mammalian genes exhibit some form of alternative splicing
Explains how approximately ==20,000== genes of the human genome can encode an estimated ==93,000== protein-encoding transcripts

# Translation
## tRNA and Ribosomes
we need:
mRNA - messenger
tRNA - transfer
- adapter molecule
- anticodon = complementary to CODON in mRNA
rRNA - ribosomal

- tRNA molecules carry amino acids to the ribosome for incorporation into a polypeptide
	- Aminoacyl-tRNA synthetases add amino acids to the acceptor stem of tRNA
	- Anticodon loop contains 3 nucleotides complementary to mRNA

![[Pasted image 20241204093944.png]]
![[Pasted image 20241204094128.png]]
![[Pasted image 20241204094134.png]]
![[Pasted image 20241204094139.png]]

http://www.viddler.com/embed/7acd69af
^^ very helpful, please rewatch!

in the ribosome, there are 3 tRNA binding sites: E site, P site, A site

# Ribosomes
RIbosomes have 2 subunits
The ribosome has multiple tRNA-binding sites
- P site - binds the tRNA at attached to the growing peptide chain
- A site - binds the tRNA carrying the next amino acid
- E site - binds the tRNA that carried the previous amino acid added

![[Pasted image 20241204094853.png]]

# Ribosome funciton
The ribosome has 2 primary functions
- Decode the mRNA
- Form peptide bonds
Peptidyl transferase
- Ennzymatic component of the ribosome
- Forms peptide bonds between amino acids
Activity of ribosomes mostly thought to be carried out by rRNAs

-----

Friday: Exam!!
- 11, 12, 13, 14, 15 (no translation or mutation)